src/api/apollo.js

```diff
@@ -1,48 +1,150 @@
 /* eslint-disable no-underscore-dangle */
-import ApolloClient from 'apollo-client';
-import { ApolloLink } from 'apollo-link';
-import {
-	IntrospectionFragmentMatcher,
-	InMemoryCache,
-	defaultDataIdFromObject,
-} from 'apollo-cache-inmemory';
+import { ApolloLink, ApolloClient, InMemoryCache } from '@apollo/client/core';
+
+import loanResolverFactory from '@/api/localResolvers/loan';
 import Auth0LinkCreator from './Auth0Link';
 import BasketLinkCreator from './BasketLink';
 import ContentfulPreviewLink from './ContentfulPreviewLink';
 import ExperimentIdLink from './ExperimentIdLink';
 import HttpLinkCreator from './HttpLink';
 import NetworkErrorLink from './NetworkErrorLink';
 import SnowplowSessionLink from './SnowplowSessionLink';
-import initState from './localState';
+// import initState from './localState';
+
+const { resolvers } = loanResolverFactory();
+const { LoanPartner, LoanDirect } = resolvers;
+
+export const typePolicies = {
+	activeLoan: {
+		merge: true
+	},
+	LoanChannel: {
+		keyFields: ['id'],
```

Comment:
I'm not sure that we need to define the `keyFields` when it's only `id`, because that's the default.